### PR TITLE
client,cmd/snap: refactor options to EnsureQuota.

### PR DESCRIPTION
**Pull request by Meulengracht**
Followup to one of the comments of my earlier PRs.

### CODE REVIEW

**Review by mvo5 — APPROVED** [review]
This change looks fine. A bit more context in the commit message might be nice, if it was e.g. the result of a previous code review suggestion it would be good to mention that. But really a nitpick.

**Comment by Meulengracht** [comment]
> This change looks fine. A bit more context in the commit message might be nice, if it was e.g. the result of a previous code review suggestion it would be good to mention that. But really a nitpick.

Good point, thanks!

File: client/quota.go [hunk 1]
```diff
@@ -74,21 +74,37 @@ type QuotaValues struct {
...
 
-// EnsureQuota creates a quota group or updates an existing group.
-// The list of snaps can be empty.
-func (client *Client) EnsureQuota(groupName string, parent string, snaps, services []string, constraints *QuotaValues) (changeID string, err error) {
+type EnsureQuotaOptions struct {
+	// Parent is used to assign a Parent quota group
+	Parent string
+	// Snaps is the Snaps that should be added to the quota group
+	Snaps []string
+	// Services is the snap Services that should be added to the quota group
```
Comment (MiguelPires) [inline client/quota.go:82]:
```suggestion
	// Services that should be added to the quota group
```

The simplified version doesn't mention "snap services". IMO the "snap" part is implicit in this context but it's subjective so it's up to you

**Review by MiguelPires — APPROVED** [review]
Minor comments about comments

**Pull request merged**